client_test.go

```diff
@@ -26,7 +25,7 @@ const TestRegistry = "quay.io/alice"
 // by the client API for those who prefer manual transmissions.
 func TestNew(t *testing.T) {
 	root := "testdata/example.com/testCreate" // Root from which to run the test
-	if err := os.MkdirAll(root, 0700); err != nil {
+	if err := os.MkdirAll(root, 0744); err != nil {
```

Comment:
Good call.  If there was a reason, I have since forgotten it and didn't add a comment to explain the magic number.  I backed test dir perms off to 700 throughout, and will loosen it if necessary)